frontend/src/components/job/info.js

```diff
@@ -23,11 +23,23 @@ class JobInfo extends Component {
                 return null
             })
 
+            let meta = [];
+            let metaTag = '<none>';
+
+            Object.keys(taskGroup.Meta || {}).forEach(function(key) {
+                meta.push(<dt key={key + 'dt'}>{key}</dt>);
+                meta.push(<dd key={key + 'dd'}>{taskGroup.Meta[key]}</dd>);
+            });
+
+            if (meta.length > 0) {
+                metaTag = <dl className="dl-horizontal">{meta}</dl>
+            }
+
             return (
                 <tr key={taskGroup.ID}>
                     <td>{taskGroup.Name}</td>
                     <td>{taskGroup.Count}</td>
-                    <td>{taskGroup.Meta || "<none>" }</td>
+                    <td>{metaTag}</td>
```

Comment:
I'd be willing to sacrifice screen estate as well, but not in the table rows. I really think this makes for a bad UX when people have to put effort into determining where the row boundaries are. I think I would be okay with showing the number of meta tags and showing them while hovering. Let me think about it bit further.